adaptation.lua

```diff
@@ -0,0 +1,110 @@
+-- Original opening comment before lua adaptation
+-- View or set cavern adaptation levels
+-- based on removebadthoughts.rb
+-- Rewritten by TBSTeun using OpenAI GPT from adaptation.rb
+
+local function print_color(color, s)
+    dfhack.color(color)
+    dfhack.print(s)
+    dfhack.color(COLOR_RESET)
+end
+
+local function usage(s)
+    if s then
+        print(s)
```

Comment:
since the message passed to `usage` is always an error message, maybe use `dfhack.printerr()` instead of `print()` here. You can make the swap without any other changes; e.g. you still will not need to manually tack on a newline.